lib/msal-node/src/utils/NodeConstants.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * http methods
+ */
+export enum method {
+    GET = 'get',
+    POST = 'post',
+}
+
```

Comment:
I don't think this needs to be named `NodeConstants`, `Constants` is probably fine.

Reply:
Will we have namespace issues as msal-common also has a Constants file? May be not, I can change this in that case

Reply:
Done